.eslintrc.js

```diff
@@ -61,7 +61,7 @@ module.exports = {
 		'import/resolver': {
 			'babel-module': { allowExistingDirectories: true },
 		},
-		'import/internal-regex': '^@/',
+		'import/internal-regex': '^#',
```

Comment:
the `@/` pattern for absolute paths was deemed confusing by NPM (because of the @scope/package pattern), and their suggested approach is this.